packages/orchestration/src/facade.js

```diff
@@ -82,27 +91,21 @@ const makeLocalChainFacade = localchain => {
 };
 
 /**
- * @template {string} C
- * @param {C} name
+ * @template {CosmosChainInfo} CCI
+ * @param {CCI} chainInfo
```

Comment:
funky.

Reply:
request?

Reply:
sorry; no... it just hand't occurred to me that a type could be parameterized by a whole structure, and not just a primitive value